behandlingskontroll/src/main/java/no/nav/foreldrepenger/tilbakekreving/behandlingskontroll/impl/BehandlingskontrollTjeneste.java

```diff
@@ -422,7 +427,17 @@ public void avsluttBehandling(BehandlingskontrollKontekst kontekst) {
     public Aksjonspunkt settBehandlingPåVent(Behandling behandling, AksjonspunktDefinisjon aksjonspunktDefinisjonIn,
                                              BehandlingStegType stegType, LocalDateTime fristTid, Venteårsak venteårsak) {
         var kontekst = initBehandlingskontroll(behandling);
-        behandling.setAnsvarligSaksbehandler(null);
+        // Nullstill ansvarlig saksbehandler dersom settes på vent utenom totrinn/beslutter
+        if (behandling.getÅpneAksjonspunkter(List.of(AksjonspunktDefinisjon.FATTE_VEDTAK)).isEmpty()) {
+            behandling.setAnsvarligSaksbehandler(null);
+        } else {
+            // Finn ut hvor dette oppstår
```

Comment:
Jeg tenker at det er god sjanse for at det skjer når vi mottar SPER-melding fra oppdragsystemet. Det ser ikke ut som det er noen sjekk i mottaket av disse meldingene som hindrer å sette på vent selv om vi er i FVED-steget.

Uansett gir det vel mening å sette på vent, vi får nok ikke iverksatt inntil kravgrunnlaget er åpnet igjen - og det er sannsynlig at kravgrunnlaget er endret når kommer ute av SPER-tilstanden